pkg/reconciler/taskrun/validate_resources.go

```diff
@@ -157,9 +157,9 @@ func wrongTypeParamsNames(params []v1beta1.Param, matrix []v1beta1.Param, needed
 	return wrongTypeParamNames
 }
 
-// missingKeysObjectParamNames checks if all required keys of object type params are provided in taskrun params.
+// MissingKeysObjectParamNames checks if all required keys of object type params are provided in taskrun params.
 // TODO (@chuangw6): This might be refactored out to support single pair validation.
```

Comment:
could you please link an issue here instead of your GH handle?

Reply:
Just deleted this todo since that functionality has been supported in the validation webhook part.